### PR TITLE
[typo](doc)flink and spark connector remove thrift compiled documents

### PR DESCRIPTION
**Pull request by DongLiang-0**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by DongLiang-0** [comment]
run buildall

**Comment by JNSimba** [comment]
Do not delete the old version, just add a version description?

**Comment by DongLiang-0** [comment]
run p0

File: docs/en/docs/ecosystem/flink-doris-connector.md [hunk 1]
```diff
@@ -54,78 +54,22 @@ Github: https://github.com/apache/doris-flink-connector
...
 
-1. Doris FE should be configured to enable http v2 in the configuration
-
-   conf/fe.conf
-
+You need to enable http v2 in the Doris FE `conf/fe.conf` file
 ```
 enable_http_server_v2 = true
 ```
+
```
Comment (JNSimba) [inline docs/en/docs/ecosystem/flink-doris-connector.md:72]: This configuration should be false by default in the new version, can it be removed?

Comment (DongLiang-0) [reply docs/en/docs/ecosystem/flink-doris-connector.md]: Thanks for your suggestion, I have deleted it.

**Comment by DongLiang-0** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**